backend/src/User/user.controller.ts

```diff
@@ -16,11 +17,21 @@ class Controller {
     }
 
     private async initInitialAdminAccount() {
-        const users = await this.userService.getAllUsers();
-        if (users.length === 0) {
+        let users = null;
+        for (let i = 0; i < 10; i++) {
+            try {
+                users = await this.userService.getAllUsers();
+                break;
+            } catch (e) {
+                logger.error(`Unable to connect to database: ${i + 1}/10`);
+                await new Promise(r => setTimeout(r, 3000));
+            }
```

Comment:
Why do we have this?

Reply:
If you start the app and the db is not ready it will try 10 times to connect instead of failing